src/cmd/src/cli/export.rs

```diff
@@ -42,14 +42,13 @@ type TableReference = (String, String, String);
 
 #[derive(Debug, Default, Clone, ValueEnum)]
 enum ExportTarget {
-    /// Corresponding to `SHOW CREATE TABLE`
-    #[default]
+    /// Export all table schemas, corresponding to `SHOW CREATE TABLE`.
     CreateTable,
-    /// Corresponding to `EXPORT TABLE`
-    #[deprecated(note = "Please use `DatabaseData` instead.")]
-    TableData,
-    /// Corresponding to `EXPORT DATABASE`
+    /// Export all table data, corresponding to `COPY DATABASE TO`.
     DatabaseData,
+    /// Export all table schemas and data at once.
```

Comment:
What's the difference with `database`?

Reply:
Schema:schemas, 
Data: data,
All: schemas&data